packages/tokens/tokens-source/brands/bring.json

```diff
@@ -26,7 +26,7 @@
 
   "button-colors": {
     "$type": "color",
-    "text": { "$value": "{ui-colors.black}" }
+    "text": { "$value": "{ui-colors.white}" }
```

Comment:
How this look? Perhaps you could take some screenshots and add them to the description in the PR?

Reply:
Reverted. It was only the Inverted button that should have white colour 😅